docs/RELEASING.md

```diff
@@ -2,24 +2,22 @@
 
 ## Prerequisites
 
-To release `cucumber-core`, you'll need:
-
-- to be a member of the core-team
-- make
-- docker
+To release `cucumber-core`, you'll need to be a member of the core team
 
 ## Releasing cucumber-core
 
 - Upgrade gems with `scripts/update-gemspec`
```

Comment:
I wonder if we really need this anymore, given that we have bots that help us upgrade dependencies nowadays?

Reply:
Renovate upgrade gemfile and gemfile.lock.
Not sure it updates gemspec.

And I think this step is interesting as it allows the developer to review what are the updates.
For example when we have updated messages to use the the json schema and the DTOs, we have noticed it as part of this step, then been able to update the changelog accordingly.

Reply:
Maybe we can move it into happening in a separate nightly action or something that proposes a PR with the changes in it. I just think it's better not to have unexpected things happening at the last minute during a release.

Reply:
Anyway, just a thought - not a blocker on merging this PR!